_posts/2017-06-03-numfocus-grant.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: NumFOCUS small grant for Python 3 support
+---
+
+We have generously been awarded a [small development grant][grant-post] by
+NumFOCUS to finally add
+Python 3 support to MDAnalysis. To do this [Richard Gowers][rg] and [Tyler Reddy][tr] will
```

Comment:
You could use the GitHub handles directly unless you explicitly want the full names there.